tpcx_bb/cluster_configuration/cluster-startup.sh

```diff
@@ -1,38 +1,39 @@
-#IB, NVLINK, or TCP
-CLUSTER_MODE=$1
+#NVLINK or TCP
+ROLE=$1
+CLUSTER_MODE="NVLINK"
 USERNAME=$(whoami)
 
 MAX_SYSTEM_MEMORY=$(free -m | awk '/^Mem:/{print $2}')M
 DEVICE_MEMORY_LIMIT="25GB"
 POOL_SIZE="30GB"
 
-# Fill in your environment name and conda path on each node
-TPCX_BB_HOME="/home/$USERNAME/shared/tpcx-bb"
+TPCX_BB_HOME=$HOME/tpcx-bb
 CONDA_ENV_NAME="rapids-tpcx-bb"
 CONDA_ENV_PATH="/home/$USERNAME/conda/etc/profile.d/conda.sh"
 
-# TODO: Unify interface/IP setting/getting for cluster startup
-# and scheduler file
-INTERFACE="ib0"
-
-# TODO: Remove hard-coding of scheduler
-SCHEDULER=$(hostname)
-SCHEDULER_FILE=$TPCX_BB_HOME/tpcx_bb/cluster_configuration/example-cluster-scheduler.json
-LOGDIR="/tmp/tpcx-bb-dask-logs/"
-WORKER_DIR="/tmp/tpcx-bb-dask-workers/"
+# Used for writing scheduler file and logs to shared storage
+LOCAL_DIRECTORY=$HOME/dask-local-directory
+SCHEDULER_FILE=$LOCAL_DIRECTORY/scheduler.json
+LOGDIR="$LOCAL_DIRECTORY/logs"
+WORKER_DIR=/tmp/$USERNAME/tpcx-bb-dask-workers/
 
 # Purge Dask worker and log directories
-rm -rf $LOGDIR/*
-mkdir -p $LOGDIR
-rm -rf $WORKER_DIR/*
-mkdir -p $WORKER_DIR
+if [ "$ROLE" = "SCHEDULER" ]; then
+    rm -rf $LOGDIR/*
+    mkdir -p $LOGDIR
+    rm -rf $WORKER_DIR/*
+    mkdir -p $WORKER_DIR
```

Comment:
Should this be limited to the scheduler only?

Reply:
The assumption (line 14) is that the cluster is using shared storage, so I believe we should only have to clear it once.

Reply:
Ah, my bad. I was under the impression that only `local_directory` was backed by a shared filesystem.